models/yang/sonic/sonic-mgmt-interface.yang

```diff
@@ -0,0 +1,79 @@
+module sonic-mgmt-interface {
+    namespace "http://github.com/Azure/sonic-mgmt-interface";
+    prefix smgmtint;
+
+    import ietf-yang-types {
+        prefix yang;
+    }
+
+    import ietf-inet-types {
+        prefix inet;
+    }
+
+    import sonic-common {
+        prefix scommon;
```

Comment:
We planned to use 'cmn' as prefix for sonic-common, can you please change.

Reply:
@Tejaswi-Goel - Can you please update this one to cmn ?

Reply:
I would prefer we use sonic-cmn as suggested in the review. "cmn" is very generic.

Reply:
Addressed